DEVELOPER.md

```diff
@@ -154,6 +152,8 @@ describe.only('your describe test', ....)
 >
 >This is useful when trying to get a specific test or subset of tests to pass.
 
+Alternatively, you can change the final part of the test script in `package.json` to point to your test file in the `./build/compile/test` directory.
```

Comment:
An example inline here would make this more clear

Reply:
I checked both of the other suggestions and those work fine, so I don't think this is even necessary.